.envrc

```diff
@@ -1 +1,3 @@
-export BROWSIR_VERSION=0.1.1
\ No newline at end of file
+export BROWSIR_VERSION=0.1.1use asdf
+source_up
```

Comment:
?

Reply:
Upsies! It's a local config refusal :grin: